### ISSUE TITLE
CNCF / KubeCon

### ISSUE THREAD

**Issue by 616b2f** [GoogleContainerTools/skaffold#8694]
Hi,

I really like the project and we use it to develop/debug for some of our services. However I was this year in KubeCon in Amsterdam and really missed any mentioning if this tool, there are tools like odo or telepresence but no skaffold. 

Any plans to introduce it in the future?

**Comment by Larswa** [comment]
Yeah I wondered about that myself. Currently looking into designing a tight inner dev workflow for services and kubernetes for a few teams at a client so deepdiving in this atm, and was surprised to only see Tilt mentioned in one talk. There is a lot of tools out there, but not a lot of good reference stories as to how to get them to play nice together. As for Skaffold it would be cool to have some docs / demoes of for instance how it plays with ... for instance ... Gefyra or Mirrord.

**Comment by aaron-prindle** [comment]
@616b2f thanks flagging this.  The Skaffold team was unable to Kubecon in Amsterdam this year unfortunately, travel logistics have been difficult for our team this year sadly.  Skaffold was actually introduced @ Kubecon in 2018:
https://youtu.be/2eAOx8E6-5Q?t=1042

but I definitely understand the sentiment of wanting more Skaffold presence and oppornuties to meet/discuss ideas around the tool, best practices, applications using it, etc..  I will discuss this with the team and try to make a concerted effort for upcoming Kubecons - will update the thread here w/ more information

**Comment by bendory** [comment]
The possibility of donating Skaffold to the CNCF came up recently [in this dicussion](https://github.com/GoogleContainerTools/skaffold/issues/2492#issuecomment-2032805645).

If you are interested in helping assemble a CNCF application to make such a donation possible, please let us know via comment here!

**Comment by shadiramadan** [comment]
I would be interested in doing so but have not yet contributed to skaffold directly although I rely on it so much!  I'll work to make a contribution first :).

**Comment by 616b2f** [comment]
> The possibility of donating Skaffold to the CNCF came up recently [in this dicussion](https://github.com/GoogleContainerTools/skaffold/issues/2492#issuecomment-2032805645).
> 
> If you are interested in helping assemble a CNCF application to make such a donation possible, please let us know via comment here!

What would that mean? I never did something like that. What does one have to do to make that happen?
